pkg/taxonomy/catalog.structs.schema.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Is this meant to be a real schema or just used for testing the tool? If the latter, perhaps it should be in a test data directory?  If it's the former then perhaps either update the description of the PR or separate into a separate PR.